pkgs/development/python-modules/geopy/2.nix

```
@@ -0,0 +1,29 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, isPy27
+, geographiclib
+}:
+
+buildPythonPackage rec {
+    pname = "geopy";
+    version = "1.20.0";
+
+    disabled = !isPy27; # only Python 2.7
+    doCheck = false; # Needs network access
+
+    # checkInputs = [ mock tox pylint ]; # not needed since we don't run checks
+    propagatedBuildInputs = [ geographiclib ];
+
+    src = fetchPypi {
+        inherit pname version;
+        sha256 = "9419bc90ee6231590c4ae7acf1cf126cefbd0736942da7a6a1436946e80830e2";
+    };
+
+    meta = with stdenv.lib; {
+        homepage = "https://github.com/geopy/geopy";
+        description = "Python Geocoding Toolbox";
+        license = licenses.mit;
+        maintainers = [ "aceus02@gmail.com" ];
```

```suggestion
        maintainers = with maintainers; [ GuillaumeDesforges ];
```

Thanks for volunteering :)
Please create an entry for yourself in https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix